### ISSUE TITLE
Access image height/width beyond EXIF tags

### ISSUE THREAD

**Issue by PanierAvide** [LeoHsiao1/pyexiv2#149]
Hello,

First thanks for your work on this library, which is super useful and we use it extensively on [Panoramax](https://panoramax.fr), especially on [a little lib](https://gitlab.com/panoramax/server/geo-picture-tag-reader) to read GPS and dates from JPEG images.

I'm wondering if there is a way to read image height/width from file itself when the dimensions are not available in EXIF tags (we used `Exif.Photo.Pixel[X|Y]Dimension` or `Xmp.GPano.CroppedAreaImage[Height|Width]Pixels` for now). In this image, not EXIF tags seem to contain this information : [example.jpg.zip](https://github.com/user-attachments/files/17202033/example.jpg.zip)

Extracted EXIF & all tags:

```
{'Exif.Image.Make': 'GoPro',
'Exif.Image.Model': 'GoPro Max',
'Exif.Image.ExifTag': '78',
'Exif.Photo.DateTimeOriginal': '2024:09:29 17:10:00',
'Exif.Photo.OffsetTimeOriginal': '+00:00',
'Exif.Photo.SubSecTimeOriginal': '293000',
'Exif.Image.GPSTag': '150',
'Exif.GPSInfo.GPSLatitudeRef': 'N',
'Exif.GPSInfo.GPSLatitude': '49/1 11/1 548764799/10000000',
'Exif.GPSInfo.GPSLongitudeRef': 'W',
'Exif.GPSInfo.GPSLongitude': '0/1 21/1 325688400/10000000',
'Exif.GPSInfo.GPSAltitudeRef': '0',
'Exif.GPSInfo.GPSAltitude': '6330/100',
'Exif.GPSInfo.GPSTimeStamp': '17/1 10/1 293000/1000000',
'Exif.GPSInfo.GPSImgDirectionRef': 'T',
'Exif.GPSInfo.GPSImgDirection': '7766/100',
'Exif.GPSInfo.GPSDateStamp': '2024:09:29',
'Exif.Photo.UserComment': 'Lavc58.134.100'}
```

Exiftool manages to detect image size :

```bash
$ exiftool -G image.jpg
[ExifTool]      ExifTool Version Number         : 12.93
...
[File]          Image Width                     : 128
[File]          Image Height                    : 96
...
[Composite]     Image Size                      : 128x96
...
```

I have no idea if this is feasible with pyexiv2 :sweat_smile: Thanks for your help :grinning: 

Best regards

**Comment by LeoHsiao1** [comment]
Hi!
I just released [v2.15.0](https://github.com/LeoHsiao1/pyexiv2/releases/tag/v2.15.0), which adds `img.get_pixel_width()` and `img.get_pixel_height()`.
It is not recommended to get the width and height from the image metadata, as the metadata may be falsified.

**Comment by PanierAvide** [comment]
Works like a charm, thanks for your fast implementation :grinning:
